File: androidfront/app/src/main/java/com/example/nero/clones/model/Clone.java
package com.example.nero.clones.model;

import java.io.Serializable;
import java.time.LocalDate;
import java.util.List;

/**
 * Created by Nero on 25/02/2018.
 */

public class Clone implements Serializable{

    private Long id;
    private String nome;
    private Integer idade;
    private String dataCriacao;
    private List<String> adicionais;

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public Integer getIdade() {
        return idade;
    }

    public void setIdade(Integer idade) {
        this.idade = idade;
    }

    public String getDataCriacao() {
        return dataCriacao;
    }

    public List<String> getAdicionais() {
        return adicionais;
    }

    public void setAdicionais(List<String> adicionais) {
        this.adicionais = adicionais;
    }

    public void setDataCriacao(String dataCriacao) {
        this.dataCriacao = dataCriacao;
    }
}

File: cyberpunk_app/app/src/main/java/com/example/mnmat/cyberpunk/models/Clone.kt
package com.example.mnmat.cyberpunk.models

import java.io.Serializable
import java.util.*

/**
 * Created by mnmat on 02/03/2018.
 */
data class Clone (var nome: String, var idade: Int, var braco_mecanico: Boolean, var esqueleto_reforcado: Boolean,
                  var sentidos_agucados: Boolean, var pele_adaptativa: Boolean, var raio_laser: Boolean) : Serializable {
     var id: Int? = null
     var data_criacao: Date? = null


}